google/cloud/firestore_v1/document.py

```diff
@@ -402,16 +403,26 @@ def get(
             create_time = None
             update_time = None
         else:
-            data = _helpers.decode_dict(document_pb.fields, self._client)
-            exists = True
-            create_time = document_pb.create_time
-            update_time = document_pb.update_time
+            batched_response: BatchGetDocumentsResponse = list(batch_response_pb)[0]
+            read_time = batched_response.read_time
+            document: Document = batched_response.found
+            # TODO: Verify this check is correct.
+            if self.id not in batched_response.missing:
```

Comment:
In the node sdk, `found` is used as the sentinel. Though missing ought to be the same? https://github.com/googleapis/nodejs-firestore/blob/1f8dc26032ef20ab874b70b1ae74988df484aec4/dev/src/index.ts#L1087

Reply:
Here, `found` is the actual document, and `missing` is the full Firestore path to the document that didn't end up existing. So there are multiple ways to infer this 🤷